src/sdk/pynni/nni/batch_tuner/batch_tuner.py

```diff
@@ -42,6 +42,13 @@ class BatchTuner(Tuner):
                              '_value': '[{...}, {...}, {...}]',
                           }
     }
+
+    Attributes
```

Comment:
Normally documented attributes are for external access.
If it's an internal state, please add an underscore at beginning of the name.